.github/workflows/ios_test.yml

```diff
@@ -1,48 +1,7 @@
 name: iOS Test
 
-on: [ pull_request ]
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref }}
-  cancel-in-progress: true
+on: [pull_request]
 
 jobs:
-  detect-changes:
-    name: Detect Changes
-    runs-on: ubuntu-latest
-    outputs:
-      # Matches all files which can affect iOS build, when changed
-      iosFiles: ${{ steps.file-changes.outputs.iosFiles }}
-      # Matches all files which can affect Android build, when changed
-      androidFiles: ${{ steps.file-changes.outputs.androidFiles }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Detect Changes
-        uses: dorny/paths-filter@v2
-        id: file-changes
-        with:
-          filters: |
-            iosFiles:
-              - "!(androidApp/**)"
-            androidFiles:
-              - "!(iosApp/**)"
```

Comment:
Tenhle job bych zachoval, at se ta workflow pusti jen kdyz jsou zmeny v iOS/shared

Reply:
To stejné je ale ve [sdíleném workflow](https://github.com/futuredapp/.github/blob/main/.github/workflows/ios-kmp-selfhosted-test.yml) 🤔